packages/radix-ui-themes/src/components/callout.parts.ts

```diff
@@ -1,3 +1,3 @@
-export { CalloutRoot as Root } from './callout';
-export { CalloutIcon as Icon } from './callout';
-export { CalloutText as Text } from './callout';
+export { CalloutRoot as Root } from './callout.js';
+export { CalloutIcon as Icon } from './callout.js';
+export { CalloutText as Text } from './callout.js';
```

Comment:
Unrelated as it came from the another PR, but is there a reason why those `.parts` files can't be combined in a single export call?

ie. for this use-case:

```ts
export {
  CalloutRoot as Root,
  CalloutIcon as Icon,
  CalloutText as Text
} from './callout.js';
```

Reply:
@penx do you happen to know whether the export style would have any effect?

Reply:
@vladmoroz sorry for the late reply, no this makes sense to change it. I think I just did it that way as it was faster the way I was doing the same copy/paste/type over several files with multi line cursors.